platform-controller/src/test/java/org/hobbit/controller/docker/FileBasedImageManagerTest.java

```diff
@@ -1,5 +1,6 @@
 package org.hobbit.controller.docker;
 
+import org.hobbit.controller.containers.FileBasedImageManager;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.